lib/streamlit/hashing.py

```diff
@@ -297,9 +297,10 @@ def to_bytes(self, obj, context=None):
         hash_stacks.push(obj)
 
         try:
-            LOGGER.debug("About to hash: %s", obj)
+            # These were super noisy so i turned them off.  --NM
+            # LOGGER.debug("About to hash: %s", obj)
             b = self._to_bytes(obj, context)
-            LOGGER.debug("Done hashing: %s", obj)
+            # LOGGER.debug("Done hashing: %s", obj)
```

Comment:
Agreed! But please put these back for now, since they're the only way we can have our users debug hashing-related problems. (We tell users to use `--global.logLevel=debug` then send us the console output)

Then after #1146 lands we can remove these.

Reply:
okie doke